scripts/tests.sh

```diff
@@ -54,7 +48,7 @@ $ETH_PATH --test -d /tmp/test &
 # is available and is ready for the unit-tests to start talking to it.
 while [ ! -S /tmp/test/geth.ipc ]; do sleep 2; done
 echo "--> IPC available."
-
+sleep 2
```

Comment:
@chriseth i thought maybe i was earlier on. Not sure if that was due to eliminating the if conditional on the eth branch. If you want to try restarting the build a number of times to see if it'll pass im okay with that.